docs/management/admin/trusted-apps.asciidoc

```diff
@@ -27,39 +31,41 @@ NOTE: You can only add a single field type value per trusted application. For ex
 
 * `Value`: Enter the hash value or file path. To add an additional value, click *AND*.
 
-* `Description`(Optional): Enter a description for the trusted application.
+. Select an option in the *Assignment* section to assign the trusted application to a specific integration policy:
++
+* `Global`: Assign the trusted application to all integration policies for {endpoint-sec}.
+* `Per Policy`: Assign the trusted application to one or more specific {endpoint-sec} integration policies. Select each policy in which you want the application to be trusted.
```

Comment:
we should add another `NOTE:` here that users are able to assign a Trusted App to 0 policies in the per policy section.  This could be useful if users want to create and review their Trusted Apps first and slowly roll them out to policies.